cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -3664,24 +3670,18 @@
 
 ;; Check for signed overflow. The only case is min_value / -1.
 ;; The following checks must be done in 32-bit or 64-bit, depending
-;; on the input type.
-(spec (trap_if_div_overflow ty x y)
```

Comment:
This spec is now stale, but the changes needed to verify the code correctly are ones we still need to upstream. Just removing for now?

Reply:
Sure -- no problem to re-add this when the rest is upstreamed.